weaviate/collection/grpc.py

```diff
@@ -230,7 +242,7 @@ def near_object(
 
         return self.__call()
 
-    def __call(self) -> List[GrpcResult]:
+    def __call(self) -> List[SearchResult]:
```

Comment:
You'd also need to change the return values in the callers

Reply:
Ah yes, missed that in the code copying!